src/dotnet-svcutil/lib/src/CodeDomFixup/EndpointSelector.cs

```diff
@@ -62,6 +62,10 @@ private static bool IsBindingSupported(Binding binding)
                     {
                         s_bindingValidationErrors.Add(string.Format(SR.BindingMessageEncodingNotSupportedFormat, wsHttpBinding.MessageEncoding, WSMessageEncoding.Text));
                     }
+                    if (wsHttpBinding.Security.Mode == SecurityMode.Message)
+                    {
```

Comment:
We don't have a second for it, but we also need to do this check for BasicHttpBinding, although there the mode type is BasicHttpSecurityMode. You need to remove the check for MessageEncoding just above this code as we now support that.

Reply:
@ZhaodongTian  chatted with Matt, it's okay we are not supporting ReliableSession, and remove ling 61 to 64; and add check for BasicHttpBinding security mode as Matt suggested.

Thanks!

Reply:
Done, I have add the code for checking BasicHttpBinding security mode